stream-chat-android/src/main/java/com/getstream/sdk/chat/view/messages/AvatarStyle.kt

```diff
@@ -0,0 +1,110 @@
+package com.getstream.sdk.chat.view.messages
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.graphics.Typeface
+import androidx.annotation.ColorInt
+import androidx.annotation.DimenRes
+import androidx.annotation.StyleableRes
+import com.getstream.sdk.chat.R
+import com.getstream.sdk.chat.style.TextStyle
+
+public data class AvatarStyle(
+    public val avatarWidth: Int = 0,
+    public val avatarHeight: Int = 0,
+    public val avatarBorderWidth: Int = 0,
+    public val avatarBorderColor: Int = 0,
+    public val avatarBackgroundColor: Int = 0,
+    public val avatarInitialText: TextStyle = TextStyle()
+) {
+    public class Builder(private val a: TypedArray, private val c: Context) {
```

Comment:
The `AvatarStyle` class, or at least its nested `Builder` can likely be `internal`.